CHANGELOG.md

```diff
@@ -31,8 +31,9 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fix typo error in Settings/Configuration [#3234](https://github.com/wazuh/wazuh-kibana-app/pull/3234)
 - Fixed fields overlap in the agent summary screen [#3217](https://github.com/wazuh/wazuh-kibana-app/pull/3217)
 - Fixed the `Visualize` button is not displaying when expanding a field in the Events sidebar [#3237](https://github.com/wazuh/wazuh-kibana-app/pull/3237)
-- Add error when add sample data fails [#3241] (https://github.com/wazuh/wazuh-kibana-app/pull/3241)
-- Fix modules are missing in the agent menu [#3244] (https://github.com/wazuh/wazuh-kibana-app/pull/3244)
+- Add error when add sample data fails [#3241](https://github.com/wazuh/wazuh-kibana-app/pull/3241)
+- Fix modules are missing in the agent menu [#3244](https://github.com/wazuh/wazuh-kibana-app/pull/3244)
+- Fixed error api/check-wazuh in no security environment [#3292](https://github.com/wazuh/wazuh-kibana-app/pull/3292)
```

Comment:
This PR fixes a feature that releases in the same revision. I think we maybe should add the fix or move the PR to the feature instead.